src/PlaywrightSharp/PlaywrightSharp.csproj

```diff
@@ -60,8 +60,6 @@
     </ContentWithTargetPath>
   </ItemGroup>
   <Target Name="EnsurePrerequisitsRan" BeforeTargets="InstallBrowsers">
-    <Error
-      Text="Playwright prerequisites are missing. Ensure you've ran `dotnet run -p ./src/tools/PlaywrightSharp.Tooling/PlaywrightSharp.Tooling.csproj -- download-drivers --basepath .`"
-      Condition="!Exists('$(MSBuildProjectDirectory)\Drivers\unix$(NativeFolder)')" />
+    <Error Text="Playwright prerequisites are missing. Ensure you've ran `dotnet run -p ./src/tools/PlaywrightSharp.Tooling/PlaywrightSharp.Tooling.csproj -- download-drivers --basepath .`" Condition="!Exists('$(MSBuildProjectDirectory)\Drivers\unix$(NativeFolder)')" />
```

Comment:
hehehehehe

Reply:
I did hint at leaving it in your PR. I mean, let's let the IDE have its (annoying) way. I can split it out into a different PR though, to stop us from having to manually remove it all the time.